tests/common/utilities.py

```diff
@@ -473,7 +473,7 @@ def is_ipv4_address(ip_address):
         return False
 
 
-def compare_crm_facts(left, right):
+def compare_crm_facts(duthost, left, right):
```

Comment:
you cannot specify the arguments this way as other scripts are also utilizing the same proc. Need the arguments to be optional for backward compatibility